packages/grid/x-data-grid-pro/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -252,6 +253,12 @@ export const useGridColumnResize = (
           `[data-field="${colDef.field}"]`,
         )!;
 
+      colGroupingElementRef.current = Array.from(
+        apiRef.current.columnHeadersContainerElementRef?.current!.querySelectorAll<HTMLDivElement>(
```

Comment:
Not a blocker for this PR, but querySelectorAll has a drawback - see https://github.com/mui/mui-x/issues/5493